src/index.js

```diff
@@ -0,0 +1,127 @@
+import './styles.css';
+import 'bootstrap';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import $ from 'jquery';
+import moment from 'moment';
+
+const dayGIF = 'https://mondrian.mashable.com/wp-content%252Fuploads%252F2013%252F04%252Fpalm-trees.gif%252Ffull-fit-in__1200x2000.gif?signature=u6Pkj2nxfqX8m9jTaqwq1Xl6iqM=&source=http%3A%2F%2Fmashable.com';
+const nightGIF = 'https://i.gifer.com/VTNI.gif';
+
+async function getCity() {
+  const cityInput = $('.city-name');
+  const cityName = cityInput.val();
+  cityInput.val('');
+
+  const celOrFahrNode = document.querySelector('.cel-or-fahr');
+  const celOrFahr = celOrFahrNode.textContent;
+  let API = '';
+
+  if (celOrFahr === 'celsius') {
+    API = `https://api.openweathermap.org/data/2.5/weather?q=${cityName}&units=metric&appid=934ea5f8bc3f198fabf59b607a2fcc71`;
+  } else {
+    API = `https://api.openweathermap.org/data/2.5/weather?q=${cityName}&units=imperial&appid=934ea5f8bc3f198fabf59b607a2fcc71`;
+  }
+
+  const response = await fetch(API);
+  const city = await response.json();
+
+  return city;
```

Comment:
- [ ] Kindly handle async fetch errors here either with a try/catch block or chaining a `.catch` to fetch requests.